homeassistant/components/switchbot_cloud/vacuum.py

```diff
@@ -90,30 +103,149 @@ async def async_set_fan_speed(self, fan_speed: str, **kwargs: Any) -> None:
     async def async_pause(self) -> None:
         """Pause the cleaning task."""
         await self.send_api_command(VacuumCommands.STOP)
+        self.async_write_ha_state()
```

Comment:
Why don't we ask the coordinator for a refresh?

Reply:
ok